### DISCUSSION TITLE
Customized URL for Causes

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#127] [Feedback]
Now the URL is being created with the value of "ID" field

This should be customizable. The first thought goes to using cause title but in some places the title would be too long for a memorable url.

**Comment by an-nadaa** [comment]
@smsalmanshahAN I avoided using slugs because if we accidentally create a cause with the same title they will clash. hence the unique id. whatever the solution we use it has to be a unique value.

another way of solving this temporary (until we setel on something) is to use a url shortener it will even give analytics on how the link was used.

in the meantime can you please give an example of how ideally a url should look like?
